husky_bringup/launch/flir_ptu/flir_ptu.launch

```diff
@@ -0,0 +1,11 @@
+<launch>
+
+  <arg name="port" default="/dev/ttyS1"/>
```

Comment:
These will not be changeable, use environment variables.  There could be cases when just a flir_ptu is used.